src/tests/unit-tests/bytecode/dune

```diff
@@ -0,0 +1,6 @@
+(tests
+ (package ocaml-protoc)
+ (names varint wrapper_encoding pbrt_array)
+ (libraries pbrt ocaml-protoc.compiler-lib)
+ (modes byte)
```

Comment:
Just curious why tests specifically in bytecode mode are necessary?

Reply:
oh most tests are run in native mode, but it's important to have at least one bytecode test to ensure there's no issue. The current stubs for `varint` encoding are different for bytecode and native (the native ones can be more optimized) so discrepancies can sneak in.

Reply:
Ah, got it! 👍